Makefile

```diff
@@ -85,13 +85,24 @@ publish-multiarch: publish-operator-multiarch publish-interceptor-multiarch publ
 
 # Development
 
-manifests: controller-gen ## Generate ClusterRole and CustomResourceDefinition objects for core componenets.
-	$(CONTROLLER_GEN) crd:crdVersions=v1 rbac:roleName=keda-http-add-on paths="./operator/..." output:crd:artifacts:config=config/crd/bases
-	$(CONTROLLER_GEN) crd:crdVersions=v1 rbac:roleName=keda-http-add-on-scaler paths="./scaler/..." output:rbac:artifacts:config=config/scaler
-	$(CONTROLLER_GEN) crd:crdVersions=v1 rbac:roleName=keda-http-add-on-interceptor paths="./interceptor/..." output:rbac:artifacts:config=config/interceptor
+generate: codegen manifests ## Generate code and manifests.
 
-verify-manifests:
-	./hack/verify-manifests.sh
+verify: verify-codegen verify-manifests ## Verify code and manifests.
+
+codegen: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+	$(CONTROLLER_GEN) object:headerFile='hack/boilerplate.go.txt' paths='./...'
+	./hack/update-codegen.sh
+
+verify-codegen: ## Verify code is up to date.
+	./hack/verify-codegen.sh
+
+manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) crd rbac:roleName='operator' webhook paths='./operator/...' output:crd:artifacts:config='config/crd/bases' output:rbac:artifacts:config='config/operator'
+	$(CONTROLLER_GEN) crd rbac:roleName='scaler' webhook paths='./scaler/...' output:rbac:artifacts:config='config/scaler'
+	$(CONTROLLER_GEN) crd rbac:roleName='interceptor' webhook paths='./interceptor/...' output:rbac:artifacts:config='config/interceptor'
```

Comment:
Should we name the roles as http-add-on-xxxxx? The naming could be confusing for cluster scoped resources. Operator as ClusterRole could be hard to know what it's for without checking the annotations/labels. WDYT @tomkerkhove ?

Reply:
This prefix is actually being added by Kustomize here: https://github.com/kedacore/http-add-on/blob/279a145940787a029b3629ab81d676a67971d249/config/default/kustomization.yaml#L9. Only the intermediate resources have this short name. I decided to do that because there was `keda-http-add-on-` and variations everywhere. This way we have simpler definitions and the prefix is now uniform.

Reply:
oh, I didn't notice it, thanks for pointing it ❤️